### PR TITLE
Add an article detailing Swift SDK specific behaviors for redirect sign-on

### PR DESCRIPTION
**Pull request by mauricesharp-okta**
## Latest preview build
12/22/23 5:57pm Eastern
https://658611d27e948f22f7d03117--reverent-murdock-829d24.netlify.app/docs/guides/mobile-swift-configure-redirect/main/

## Sizing
Medium (well, Medium Small)

## Description:
 **What's changed?**
Added an article that details the redirect authentication method behaviors that are specific to the Swift SDK. Includes a new article with an image, plus changes to add the article.

…/docs/guides/mobile-swift-configure-redirect
- Folder for new article

…/mobile-swift-configure-redirect/index.md
- Metadata for new article

…/.viewpress/public/img/mobile-sdk/mobile-redirect-alert.png
- Image used in new article

specificlinks.md
- Added link to article in main mobile redirect login article snippet

getuserinfo.md
- Added missing word to mobile redirect login article snippet

@okta/vuepress-theme-prose/navbar.const.js
- Added new article to left navbar

…/guides/index.md
- Added new article to list of guides

- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-670144](https://oktainc.atlassian.net/browse/OKTA-670144)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/3cbdeabd0b67bbc6cf8de37a16c3e96a712c17ce/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | 74 | 29 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9ff6352c-adff-4708-bb5d-2e2e374310d5) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/3cbdeabd0b67bbc6cf8de37a16c3e96a712c17ce/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Findex.md) | 100 | 6 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/612a57cb-d847-46fe-8ead-9567130a4474) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/3cbdeabd0b67bbc6cf8de37a16c3e96a712c17ce/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Fmain%2Findex.md) | 83 | 13 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0a8a56cf-cf62-4022-a216-3e195343a7b4) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md](https://github.com/okta/okta-developer-docs/blob/3cbdeabd0b67bbc6cf8de37a16c3e96a712c17ce/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fgetuserinfo.md) | 100 | 6 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94b9e8f8-5782-444e-b8a0-72c2ed3205b6) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/specificlinks.md](https://github.com/okta/okta-developer-docs/blob/3cbdeabd0b67bbc6cf8de37a16c3e96a712c17ce/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fspecificlinks.md) | 90 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b3f5dda5-b6f2-4b4d-9b07-c2efe3cc20a6) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 5 of 5 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.9692aaa2-d634-4395-a70b-22fb5dc4b42e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/6212bf41bc955f848c33da1b22a823fc1af5c8ab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | 74 | 29 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f4961d86-9328-4192-b8a1-928eed2ba631) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/6212bf41bc955f848c33da1b22a823fc1af5c8ab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Findex.md) | 100 | 6 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ca10ad36-3ae3-446a-8d4c-ed86a2108ffc) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/6212bf41bc955f848c33da1b22a823fc1af5c8ab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Fmain%2Findex.md) | 90 | 10 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b11d8e4f-9e25-4a3f-a31c-b10bbc006169) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md](https://github.com/okta/okta-developer-docs/blob/6212bf41bc955f848c33da1b22a823fc1af5c8ab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fgetuserinfo.md) | 100 | 6 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/688e4ae4-b587-4cbb-8fa7-9bc6372b804b) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/specificlinks.md](https://github.com/okta/okta-developer-docs/blob/6212bf41bc955f848c33da1b22a823fc1af5c8ab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fspecificlinks.md) | 90 | 8 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/07e54ce1-7507-47b7-9892-9bb276700aa4) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 5 of 5 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.915b3db0-35fd-4008-b155-84a42729749a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+| `false`            | Yes             | Yes | No                     |
+
+## Sign out the user
+
+There are two steps to sign out a user completely when `ephemeralSession` is set to `false`:
+
+1. Call the `signOut(from:credential:options:)` or `signOut(from:credential:options:completion:)` function of the `WebAuthentication` class to invalidate the browser window session.
+2. Call the `remove()` function of the `Credential` class to remove any existing authentication tokens.
+
+If you don't do this then when the user next authenticates the token may be updated without requiring any factors. The user experiences this as an immediate authentication to the last account from which they signed-out.
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md:42]:
```suggestion
If you don't do this then when the user next authenticates the token may be updated without requiring any factors. The user experiences this as an immediate authentication to the last account from which they signed out.
```

**Review by grahamsmith-okta — CHANGES_REQUESTED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/10415ffc71533eb03327fd01b30a069563839d62/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [74](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0af7ecd9-0463-415d-a33e-f6fd562295d1) | 39  | 48  | 76  | 86  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/10415ffc71533eb03327fd01b30a069563839d62/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/efc0301d-5498-4265-ac09-84ae817964ac) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/10415ffc71533eb03327fd01b30a069563839d62/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Fmain%2Findex.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5845d1bb-4169-4dac-ab73-101103c60357) | 80  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md](https://github.com/okta/okta-developer-docs/blob/10415ffc71533eb03327fd01b30a069563839d62/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fgetuserinfo.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/77f8c901-32be-4d17-ab97-44159d7c0051) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/specificlinks.md](https://github.com/okta/okta-developer-docs/blob/10415ffc71533eb03327fd01b30a069563839d62/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fspecificlinks.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b2ddc906-dcfb-480a-9cc0-26b3a1277f90) | 32  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.42417f80-3a7f-46dc-a61c-01456d90f013?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/da3846cfa7a02f08f3fa98fdb7bbf84a8ec51dd7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/90433c00-db31-45aa-aa40-81949c825cc2) | 50  | 55  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/da3846cfa7a02f08f3fa98fdb7bbf84a8ec51dd7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/38c4c673-36a3-4c1d-8421-4c5e6aa7f8f4) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/da3846cfa7a02f08f3fa98fdb7bbf84a8ec51dd7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Fmain%2Findex.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/847a98b4-1941-4a3e-bb18-b7c5d17de262) | 80  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md](https://github.com/okta/okta-developer-docs/blob/da3846cfa7a02f08f3fa98fdb7bbf84a8ec51dd7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fgetuserinfo.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/03e45bec-39d5-41ba-859f-a375cefcba65) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/specificlinks.md](https://github.com/okta/okta-developer-docs/blob/da3846cfa7a02f08f3fa98fdb7bbf84a8ec51dd7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fspecificlinks.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5418bf62-7711-49b5-8e7b-b0e40eeb1168) | 32  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.8c8a0a4e-3925-4471-b849-6bd166f38a99?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — DISMISSED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/14de9e7dd4650274072b45a7ba3dca55d35412b8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/70d8236f-8adf-49f7-a446-3087dc268359) | 50  | 55  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/index.md](https://github.com/okta/okta-developer-docs/blob/14de9e7dd4650274072b45a7ba3dca55d35412b8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8834f0d1-c636-4cb1-ae8c-e2ee548d306c) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/mobile-swift-configure-redirect/main/index.md](https://github.com/okta/okta-developer-docs/blob/14de9e7dd4650274072b45a7ba3dca55d35412b8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmobile-swift-configure-redirect%2Fmain%2Findex.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d3fc56c0-bf12-44f8-8b6d-54a2070858da) | 80  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md](https://github.com/okta/okta-developer-docs/blob/14de9e7dd4650274072b45a7ba3dca55d35412b8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fgetuserinfo.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c7413720-9ad2-418f-aa96-bf7d01d65f8b) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/specificlinks.md](https://github.com/okta/okta-developer-docs/blob/14de9e7dd4650274072b45a7ba3dca55d35412b8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsign-into-mobile-app-redirect%2Fmain%2Fios%2Fspecificlinks.md) | [90](https://okta.acrolinx.cloud/api/v1/checking/scorecards/db9af2f9-b113-4273-b6b5-51c69845d9cc) | 32  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.6ec23a50-f53f-4b8a-bf0e-6628150526c3?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by mauricesharp-okta** [comment]
Too much messing with build process and other things... I'm going to close this PR and open a new one

**Pull request closed without merging**